app/navigation/OtherIndividualNavigator.scala

```diff
@@ -0,0 +1,90 @@
+    case NonUkAddressPage => controllers.routes.PassportDetailsYesNoController.onPageLoad(mode)
+    case WhenIndividualAddedPage => controllers.routes.CheckDetailsController.onPageLoad()
+  }
+
+  private def yesNoNavigation(mode: Mode): PartialFunction[Page, UserAnswers => Call] = {
+    case DateOfBirthYesNoPage => ua =>
+      yesNoNav(ua, DateOfBirthYesNoPage, controllers.routes.DateOfBirthController.onPageLoad(mode), controllers.routes.NationalInsuranceNumberYesNoController.onPageLoad(mode))
+    case NationalInsuranceNumberYesNoPage => ua =>
+      yesNoNav(ua, NationalInsuranceNumberYesNoPage, controllers.routes.NationalInsuranceNumberController.onPageLoad(mode), controllers.routes.AddressYesNoController.onPageLoad(mode))
+    case LiveInTheUkYesNoPage => ua =>
+      yesNoNav(ua, LiveInTheUkYesNoPage, controllers.routes.UkAddressController.onPageLoad(mode), controllers.routes.NonUkAddressController.onPageLoad(mode))
+    case PassportDetailsYesNoPage => ua =>
+      yesNoNav(ua, PassportDetailsYesNoPage, controllers.routes.PassportDetailsController.onPageLoad(mode), controllers.routes.IdCardDetailsYesNoController.onPageLoad(mode))
+  }
+
+  private def navigationWithCheck(mode: Mode): PartialFunction[Page, UserAnswers => Call] = {
+    mode match {
+      case NormalMode => {
+        case NationalInsuranceNumberPage | PassportDetailsPage | IdCardDetailsPage  => _ =>
+          controllers.routes.WhenIndividualAddedController.onPageLoad()
+        case AddressYesNoPage => ua =>
+          yesNoNav(ua, AddressYesNoPage, controllers.routes.LiveInTheUkYesNoController.onPageLoad(mode), controllers.routes.WhenIndividualAddedController.onPageLoad())
+        case IdCardDetailsYesNoPage => ua =>
+          yesNoNav(ua, IdCardDetailsYesNoPage, controllers.routes.IdCardDetailsController.onPageLoad(mode), controllers.routes.WhenIndividualAddedController.onPageLoad())
+      }
+      case CheckMode => {
+        case NationalInsuranceNumberPage | PassportDetailsPage | IdCardDetailsPage => ua =>
+          checkDetailsRoute(ua)
+        case AddressYesNoPage => ua =>
+          yesNoNav(ua, AddressYesNoPage, controllers.routes.LiveInTheUkYesNoController.onPageLoad(mode), checkDetailsRoute(ua))
+        case IdCardDetailsYesNoPage => ua =>
+          yesNoNav(ua, IdCardDetailsYesNoPage, controllers.routes.IdCardDetailsController.onPageLoad(mode), checkDetailsRoute(ua))
+      }
+    }
+  }
+
+  def yesNoNav(ua: UserAnswers, fromPage: QuestionPage[Boolean], yesCall: => Call, noCall: => Call): Call = {
+    ua.get(fromPage)
+      .map(if (_) yesCall else noCall)
+      .getOrElse(controllers.routes.SessionExpiredController.onPageLoad())
+  }
+
+  def checkDetailsRoute(answers: UserAnswers): Call = {
+    answers.get(IndexPage) match {
+      case None => controllers.routes.SessionExpiredController.onPageLoad()
+//      case Some(x) =>
+//        controllers.routes.CheckDetailsController.renderFromUserAnswers(x)
```

Comment:
For now could this be
`case _ => controllers.routes.FeatureNotAvailableController.onPageLoad()`
?